src/main/java/com/bravo/user/model/filter/UserFilter.java

```diff
@@ -7,9 +7,40 @@
 @Data
 public class UserFilter {
 
+  public Set<String> getIds() {
+    return ids;
+  }
+
   private Set<String> ids;
   private Set<String> firstNames;
   private Set<String> lastNames;
   private Set<String> middleNames;
+  private Set<Integer> phoneNumbers;
+
+  public Set<String> getFirstNames() {
```

Comment:
These getters/setters are not necessary since the [Lombok](https://projectlombok.org/) `@Data` annotation provides them.